sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceInspectionResult.java

```diff
@@ -1,4 +1,15 @@
 package com.microsoft.hydralab.performance;
 
+import java.io.File;
+
 public class PerformanceInspectionResult {
```

Comment:
Effective java: always overwrite equals and hashcode methods for a POJO.

Reply:
And toString for better debuggability.

Reply:
Tip: this can be easily done via IDEA IDE's `generate method`

Reply:
Thank you for the advice. As the design of `PerformanceInspectionResult` has not been determined, we will add the methods later. I added TODOs to the codes.